kaldb/src/main/java/com/slack/kaldb/metadata/service/ServicePartitionMetadata.java

```diff
@@ -85,4 +88,31 @@ public static Metadata.ServicePartitionMetadata toServicePartitionMetadataProto(
         .addAllPartitions(metadata.partitions)
         .build();
   }
+
+  /*
+   get partitions that match on two criteria
+   1. index name
+   2. partitions that have an overlap with the query window
+  */
+  @VisibleForTesting
```

Comment:
Since this isn't visible just for testing anymore, we can go ahead and remove this annotation.